scss/4-elements/_lists.scss

```diff
@@ -10,42 +9,80 @@ li,
   }
 }
 
-ol,
-.cads-list-ordered {
+%cads-list-ordered {
   padding: 0 0 0 $cads-spacing-5;
   margin: 0 0 $cads-spacing-4 0;
+
+  li {
+    @extend %cads-list-item;
+  }
 }
 
-ul,
-.cads-list-unordered {
+%cads-list-unordered {
   padding: 0 0 0 $cads-spacing-4;
   margin: 0 0 $cads-spacing-4 0;
   list-style-type: disc;
 
-  &.cads-list-unordered__inline {
-    padding: 0;
-    margin: 0 0 $cads-spacing-4 0;
-    list-style-type: none;
+  li {
+    @extend %cads-list-item;
+  }
+}
+
+%cads-list-unordered-inline {
+  padding: 0;
+  margin: 0 0 $cads-spacing-4 0;
+  list-style-type: none;
+
+  li {
+    display: inline-block;
+    margin-right: 1rem;
+  }
+
+  // Reverse margins if this is right aligned
+  .cads-align-right & {
```

Comment:
Is this only used in the header? If so maybe we should inline these styles there?